download-build-run-locale-screenshots.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+HOME=""
+REPO="https://github.com/mozilla-mobile/firefox-tv.git"
+CLONE_DIRECTORY="$HOME/firefox-tv"
+SCREENSHOTS_DIR="$HOME/firefox-tv/fastlane"
+CONFIG="$HOME/Screengrabfile"
+#DROPBOX_DIR="/Users/firefoxtv/Dropbox"
+
+# export target Amazon Fire TV serial (AFTT 5.1.1)
+export ANDROID_SERIAL=""
+
+# set SDK location
+export ANDROID_HOME=/Users/firefoxtv/Library/android/sdk
+
+# wake device and unlock
+adb -s $ANDROID_SERIAL shell input keyevent 82
+
+# disable system animations
+adb -s $ANDROID_SERIAL shell settings put global window_animation_scale 0
+adb -s $ANDROID_SERIAL shell settings put global transition_animation_scale 0
+adb -s $ANDROID_SERIAL shell settings put global animatior_duration_scale 0
+
+# clone repository
+cd $HOME
+
+if [ -d "$CLONE_DIRECTORY" ]; then
+ cd "$CLONE_DIRECTORY"
+ git pull origin master
+else
+ git clone "$REPO" "$CLONE_DIRECTORY"
+ cd "$CLONE_DIRECTORY"
+fi
+
+# build debug app
+./gradlew clean app:assembleAmazonWebViewDebug
+
+# build test app
+./gradlew app:assemblAndroidTest
+
+# uninstall existing apps
+./gradlew uninstallAmazonWebviewDebug uninstallAmazonWebviewDebugAndroidTest
+
+# install apps
+./gradlew installAmazonWebviewDebug installAmazonWebviewDebugAndroidTest
+
+# copy fastlane config from desktop
+if [ ! -f "$CONFIG" ]; then
+ echo "Fastlane config missing"
+ exit 1
+else
+ cp "$CONFIG" "$CLONE_DIRECTORY"
+fi
+
+# run screenshots
+fastlane screengrab run
+
+# compress images and rename report
+cd "$SCREENSHOTS_DIR"
+find . -name "*.png" | xargs sips -Z 640
+
+cd "$SCREENSHOTS_DIR/metadata/android"
+find . -name "*.html" -exec mv {} "index.html" \;
+
+# upload
```

Comment:
let's make it 'optional' and describe that this line is to upload to dropbox/cloud sync folder